src/xcode/ENA/ENA/Resources/Assets/Assets.xcassets/Certificates/EUStarsGroup.imageset/Contents.json

```diff
@@ -1,7 +1,7 @@
 {
   "images" : [
     {
-      "filename" : "lightStarsNew.pdf",
+      "filename" : "Stars_half.pdf",
```

Comment:
I don't think this can be the right solution to cut off the circle of stars to half a circle. Have you consulted UX if half the circle of stars is enough? 
We might have other situations where we can see more than the half circle. This solution will lead to a missing star images in the layer.

Reply:
Yes, me and Nick had a discussion with UX and they are onboard on this solution.